exercises/practice/complex-numbers/generate_tests.exs

```diff
@@ -0,0 +1,158 @@
+# Generates test cases and basic template, which will need to be adapted by hand
+# Pass the name of the exercise (e. g, "complex-numbers") as an argument
```

Comment:
Yes, it should definitely be in `bin`, I just left it here for now for discussion. 
I will probably expand it a bit to generate all of the other files, might as well,  and rename it `bootstrap_practice_exercise.exs` or something to be more explicit. If it can serve as a basis for a better test generator later, that would be good.